### DISCUSSION TITLE
How to list all loaded plugins?

### DISCUSSION THREAD

**Discussion by kflak** [lumen-oss/rocks.nvim#348] [Q&A] [answered]
Hi there,

Is there a way to get an overview over which plugins are active at any given time? I have some issues with a plugin where I suspect some keymappings are being stomped on by something else, but I can't quite figure out what that other plugin could be... Using rocks-config to set it up.

**Comment by mrcjkb** [comment, marked as answer]
Hey :wave: 

Neovim has built-in support for this.

- `:scriptnames` lists all sourced script names in the order they were sourced.
- `:map` lists keymaps that have been created.

**Comment by kflak** [comment]
Cool, thanks!
